crates/storage/libmdbx-rs/src/txn_manager.rs

```diff
@@ -371,11 +356,17 @@ mod read_transactions {
                 sleep(MAX_DURATION + READ_TRANSACTIONS_CHECK_INTERVAL);
 
                 assert!(!read_transactions.active.contains_key(&tx_ptr));
-                assert!(read_transactions.aborted.contains(&tx_ptr));
```

Comment:
please add a check that timed out transactions contains the tx

Reply:
yep agreed, added